css/css-values/percentage-rem-low.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>REM value with :root set to low percentage</title>
+<link rel="author" title="Jon Ege Ronnenberg" href="mailto:jon.ronnenberg+wpt@gmail.com">
+<link rel="help" href="https://www.w3.org/TR/css-values-3/#rem">
```

Comment:
```suggestion
<link rel="help" href="https://drafts.csswg.org/css-values/#rem">
```

Reply:
(Specs under /TR/ are not always the most up-to-date, so to avoid anyone reading an outdated spec we link to the editor's drafts as much as possible. But we haven't gone and changed links in all existing tests.)

Reply:
- As I said on IRC - my understanding is that a ref to a Candidate Release is more appropriate than a ref to Draft note, that can change. If that is wrong.. ok, I'm out my knowledge zone. I'll change it to drafts.csswg.